timeboost-sequencer/src/decrypt.rs

```diff
@@ -363,10 +420,13 @@ impl Worker {
                 cmd = self.rx.recv() => match cmd {
                     Some(Command::Dkg(incl)) => {
                         let round = incl.round();
-                        let bundles = incl.dkg_bundles();
-                        info!("received: {} dkg bundles in round: {}", bundles.len(), round);
-                        // TODO: ACS state machine
-                        continue;
+                        trace!(%node, %round, "dkg request");
+                        match self.on_dkg_request(incl).await {
+                            Ok(()) => {}
+                            Err(DecrypterError::End(end)) => return end,
+                            Err(err) => warn!(node = %self.label, %round, %err, "error on dkg request")
```

Comment:
the two err cases are already handled by Decrypter's main loop? can directly `self.on_dkg_request(incl).await?` here